cli/azd/pkg/environment/manager.go

```diff
@@ -317,6 +320,15 @@ func (m *manager) Reload(ctx context.Context, env *Environment) error {
 // ensureValidEnvironmentName ensures the environment name is valid, if it is not, an error is printed
 // and the user is prompted for a new name.
 func (m *manager) ensureValidEnvironmentName(ctx context.Context, spec *Spec) error {
+	exampleText := ""
```

Comment:
Likely doesn't matter in this case, because the number of examples is small, but [`strings.Builder`](https://pkg.go.dev/strings#Builder) can also be useful in cases like this since it is "mutable".